gremlin-dotnet/src/Gremlin.Net/Gremlin.Net.csproj

```diff
@@ -67,7 +67,7 @@ NOTE that versions suffixed with "-rc" are considered release candidates (i.e. p
     <RepositoryUrl>https://github.com/apache/tinkerpop</RepositoryUrl>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>    
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <EnablePackageValidation>true</EnablePackageValidation>
+    <EnablePackageValidation>false</EnablePackageValidation>
```

Comment:
Please revert this change. Package validation ensures that we don't accidentally add breaking changes without being aware of it.
In this case, we accept the breaking change of renaming the generic type parameter from `E` to `E_` of `GraphTraversal`. So, we can silence the warning for this change. You can do this by adding the following entry to the file `CompatibilitySuppressions.xml` in `gremlin-dotnet/src/Gremlin.Net`:

```xml
<!-- Renamed generic type parameter name in GraphTraversal -->
<Suppression>
  <DiagnosticId>CP0001</DiagnosticId>
  <Target>T:Gremlin.Net.Process.Traversal.GraphTraversal`2</Target>
  <Left>lib/netstandard2.0/Gremlin.Net.dll</Left>
  <Right>lib/netstandard2.0/Gremlin.Net.dll</Right>
  <IsBaselineSuppression>true</IsBaselineSuppression>
</Suppression>
```

behind the other two suppression entries there.

I realize though that this compatibility checking and how to deal with its warnings isn't easy to understand, especially for non-.NET developers who just want to modify Gremlin and therefore also have to make changes in .NET. So, I'm wondering whether we should keep the feature at all. Nevertheless, that shouldn't be part of this PR, so please add the suppression for now and reenable the validation here.

Reply:
@FlorianHockmann  Should I add all renamed classes and methods to `CompatibilitySuppressions.xml`?

How to make exclusion for method with generic param like `Gremlin.Net.Process.Traversal.GraphTraversalSource.Inject<S>(params S[])`?

Reply:
You can generate the file automatically:

 ```bash
dotnet pack gremlin-dotnet/src/Gremlin.Net /p:GenerateCompatibilitySuppressionFile=true
```

or apparently also if you temporarily add this to the Gremlin.Net.csproj:

```xml
<GenerateCompatibilitySuppressionFile>true</GenerateCompatibilitySuppressionFile>
```

as described [here](https://docs.microsoft.com/en-us/dotnet/fundamentals/package-validation/overview#suppress-compatibility-errors).

This will remove the comments that we currently have in the file, but I guess those aren't that helpful any way so it shouldn't be a problem.

Reply:
updated in https://github.com/apache/tinkerpop/pull/1808/commits/5028d7a6078f828a5b8d69426464e67bb5853f22